### PR TITLE
Remote Control Center solution

### PR DESCRIPTION
**Pull request by domcatalano**
*Issue #, if available:*

*Description of changes:*
Adding Remote Control Center solution to enable centralized management of prompts, translations, and routing configurations in DynamoDB

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: python/remote-control-center/README.md [hunk 1]
```diff
@@ -0,0 +1,123 @@
...
+3. Run the following commands
+   1. ` sam build `
+   2. ` sam package --s3-bucket <YOUR_S3_BUCKET> --output-template-file packaged.yaml `
+   3. ` sam deploy --template-file packaged.yaml --stack-name remote-control-center   --capabilities CAPABILITY_IAM  `
+4. Wait while the template deploys the following resources
+   1. ConfigTable - The Amazon DynamoDB table used to store configurations
+   2. HydrateConfigTableLambda - An AWS Lambda function used to populate values into the config table
+   3. RetrieveConfigLambda - An AWS Lambda function to be invoked from within a contact flow
+   4. HydrateConfigTableLambdaRole - An AWS IAM Role for HydrateConfigTableLambda
+   5. RetrieveConfigLambdaRole - An AWS IAM Role for RetrieveConfigLambda
```
Comment (efh365) [inline python/remote-control-center/README.md:39]: Can you add a description as to what permissions these IAM roles have?

i.e. "An AWS IAM Role for the HydrateConfigTableLambda that allows the lambda to make reads and writes to DynamoDB"

File: python/remote-control-center/README.md [hunk 2]
```diff
@@ -0,0 +1,123 @@
...
+1. Navigate to the Amazon Connect console
+2. Select your instance (or create a new one)
+3. Within the Amazon Connect service page under Contact Flows, give your instance permission to the RetrieveConfigLambda function
+4. Log into the instance
+5. Under routing, select contact flows and select create new contact flow
+6. Import the example remote-Sample.json contact flow
+7. For the two Invoke AWS Lambda function blocks, update the target lambda function to the RetrieveConfigLambda function
+8. Save, publish, and route a phone number to the contact flow
+9. Test the contact flow for each of the available languages.  After testing option 4 (Italian) notice how the prompts were automatically translated into Italian and then stored in the DynamoDB table
+10. Set the HOT_MESSAGE_FLAG configuration to 1 and test.  Then set it to 2
```
Comment (efh365) [inline python/remote-control-center/README.md:68]: I assume this flag is set in the contact flow?

File: python/remote-control-center/README.md (hunk 1 above)
Comment (domcatalano) [reply python/remote-control-center/README.md]: Added verbiage around permissions; appended " that allows the function to read and write to the the ConfigTable as well as translate text through Amazon Translate" to lines.

File: python/remote-control-center/README.md (hunk 2 above)
Comment (domcatalano) [reply python/remote-control-center/README.md]: Added information on how to change the item in the dynamodb table:
"10. Navigate to the Amazon DynamoDB service and select the ConfigTable.  Edit the item for the HOT_MESSAGE_FLAG configuration and set it to 1, save the item, and test.  Then try setting it to 2.  Notice how the behavior of the contact flow changed without having to make any modifications to the contact flow itself.  When the flag is equal to 1, the HOT_MESSAGE configuration is played out before entering the main menu.  When the flag is set to 2, the EMERGENCY_MESSAGE configuration is played out, and the caller is disconnected."

File: python/remote-control-center/packaged.yaml [hunk 3]
```diff
@@ -0,0 +1,66 @@
...
+Globals:
+  Function:
+    Timeout: 30
+    Runtime: python3.7
+    Handler: lambda_function.lambda_handler
+Resources:
+  RetrieveConfigLambda:
+    Type: AWS::Serverless::Function
+    Properties:
+      CodeUri: s3://cf-templates-shk3r44gr6qq-us-east-1/ca69335269d13c8dd4563af2b4c5b9e1
```
Comment (ranman) [inline python/remote-control-center/packaged.yaml:13]: This is an arbitrary s3 bucket - can we either reference from SAM or with a less arbitrary bucket?

File: python/remote-control-center/packaged.yaml [hunk 4]
```diff
@@ -0,0 +1,66 @@
...
+          ConfigTable:
+            Ref: ConfigTable
+          TablePrimaryKey: CollectionId
+          TableSortKey: ConfigId
+          DefaultConfig: NA
+  HydrateConfigTableLambda:
+    Type: AWS::Serverless::Function
+    Properties:
+      Timeout: 900
+      CodeUri: s3://cf-templates-shk3r44gr6qq-us-east-1/3e5ebe3a090becd097f14c89f277076b
```
Comment (ranman) [inline python/remote-control-center/packaged.yaml:35]: same as above

File: python/remote-control-center/packaged.yaml [hunk 5]
```diff
@@ -0,0 +1,66 @@
+AWSTemplateFormatVersion: '2010-09-09'
+Transform: AWS::Serverless-2016-10-31
+Description: An AWS Serverless Specification template describing your function.
+Globals:
+  Function:
+    Timeout: 30
+    Runtime: python3.7
```
Comment (ranman) [inline python/remote-control-center/packaged.yaml:7]: any reason for not 3.8?

Comment (domcatalano) [reply python/remote-control-center/packaged.yaml]: at the time of dev, there was python3.7 wasn't supported.  Updated with 3.8

File: python/remote-control-center/packaged.yaml (hunk 3 above)
Comment (domcatalano) [reply python/remote-control-center/packaged.yaml]: packaged.yaml is part of sam deployment for an account, removing from repo

**Pull request merged**